test/example/qt.erl

```diff
@@ -0,0 +1,402 @@
+-module(qt).
+
+-export([s/0, c/0]).
+
+-include("../../include/quicer.hrl").
+
+%% Usage: run qt:s() in one window, and qt:c() in another window.
+%% Playaround with the client and do things like:
+
+%% Start server as: erl -pa ../../_build/default/lib/quicer/ebin -s qt s
+%% Start the client as:
+%% $ erl -pa ../../_build/default/lib/quicer/ebin
+%% Erlang/OTP 25 [erts-13.0.4] [source] [64-bit] [smp:12:12] [ds:12:12:10] [async-threads:1] [jit:ns]
+%%
+%% Eshell V13.0.4  (abort with ^G)
+%% 2> qt:c().
+%% -->
+
+
+
+%% 1> qt:c().
+%% --> connect.
+%% Connection # 1
+%% --> {stream, 1}.
+%% Sent negotiate data
+%% Stream # 1
+%% --> {ping, 1}.
+%% Got pong 1
+%% ok
+%% --> {close_stream, 1}.
+%% ok
+%% -->
+
+%% Also interesting is to experiment with various variants of CTL-Z and
+%% CTL-C on both the server and the client.
+
+
+
+%% The server
+
+-define(INTERVAL, 3000).
+
+s() ->
+    application:ensure_all_started(quicer),
+    Port = 4567,
+    LOptions = [ {cert, "cert.pem"}
+               , {key,  "key.pem"}
+               , {handshake_idle_timeout_ms, 3 * ?INTERVAL}
+               , {keep_alive_interval_ms, ?INTERVAL}
+               , {alpn, ["sample"]}
+               , {idle_timeout_ms, 3 *?INTERVAL}
+               , {peer_bidi_stream_count, 64000}
+               ],
+    proc_lib:spawn_link(fun() ->
+                                {ok, L} = quicer:listen(Port, LOptions),
+                                listener(L)
+                        end).
+
+
+listener(L) ->
+    Top = self(),
+    P = proc_lib:spawn_link(
+          fun() ->
+                  acceptor(Top, L)
+          end),
+    receive {P, more} -> ok end,
+    listener(L).
+
+
+acceptor(Top, L) ->
+    io:format("Call accept ~p\n", [self()]),
+    {ok, Conn} = quicer:accept(L, [], infinity),
+
+    Top ! {self(), more},
+
+    io:format("accept -> ~p\n",[Conn]),
+
```

Comment:
FYI, maybe call `quicer_nif:async_accept_stream` here, force we have stream acceptor in the wait list before handshake is done.

Because there could be a race here that when new stream is created by remote (client) (after handshake) we dont have acceptor yet as in L89 hasn't been called. The quicer will select connection owner as the stream owner as fallback.